package.json

```diff
@@ -5,10 +5,10 @@
   "homepage": "https://ebi-uniprot.github.io/uniprot-website/",
   "license": "MIT",
   "scripts": {
-    "start": "webpack-dev-server --env.PUBLIC_PATH=/ --mode development",
-    "build:prod": "webpack --env.PUBLIC_PATH=/ --mode production",
-    "build:staging": "webpack --env.PUBLIC_PATH=/uniprot-website/ --mode production",
-    "build:dev": "webpack --env.PUBLIC_PATH=/ --mode development",
+    "start": "webpack serve --env PUBLIC_PATH=/ --mode development",
```

Comment:
>Why was webpack-dev-server removed from this line? If not needed we can remove from the dependencies

It is now included in webpack CLI, so we're using it, but not directly anymore (as per webpack 5 upgrade recommendations that I was following). I tried to remove it but it complained it wasn't there. Not a big fan of dependencies that we are not using ourselves directly, but it looks like that's how it works now.

>Noticed that it won't open the next available port but instead raises an error on 8080

So apparently that's a bug on the side of webpack-dev-server. Fixed with the next major version (see https://github.com/webpack/webpack-dev-server/releases/tag/v4.0.0-beta.0 changelog) but not sure when it will be released.
This can be avoided now by running `yarn run start --port 8081` for example, even if I agree it's not nice. Would you prefer we upgrade to the beta version of the dev server now?

Reply:
Updated now